website/www/site/content/en/documentation/sdks/yaml-providers.md

```diff
@@ -133,12 +133,14 @@ the implementation in terms of other YAML transforms.
         properties:
           n: {type: integer}
       body:
-        type: MapToFields
-        config:
-          language: python
-          append: true
-          fields:
-            power: "element ** {{n}}"
+        type: chain
```

Comment:
should this be same as sdks/python/apache_beam/yaml/yaml_provider_unit_test.py?

Reply:
I updated this doc to show something that works with the currently released version of beam. The fix in this PR is needed for the non-chain case to work.

Reply:
ah, I see.